src/cljs/main/broadfcui/common/style.cljs

```diff
@@ -89,7 +89,8 @@
 
 (defn create-message-well [message]
   [:div {:style {:textAlign "center" :backgroundColor (:background-light colors)
-                 :padding "1em 0" :borderRadius 8}}
+                 :padding "1em 0" :borderRadius 8}
+         :data-test-id "empty-message"}
```

Comment:
Either `data-test-id` could be passed in to `create-message-well` or the ID here could have a more generic name... then tests can grab it and check the text contents.